### PR TITLE
added results and rules page

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Hello everyone done with the Results and rules page for the Quiz Mini Project
Live UI for Rules -https://deploy-preview-3--piston-quizes.netlify.app/pages/rules.html
Live UI for https://deploy-preview-3--piston-quizes.netlify.app/pages/results.html

### CODE REVIEW

File: css-files/rules-results.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.car-icon-list{}
```
Comment (sainath7878) [inline css-files/rules-results.css:1]: remove if not required.

Comment (Dev-AkshatGupta) [reply css-files/rules-results.css:1]: Removed thanks for Reviewing

File: pages/rules.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+                </ul>
+                <a href="" class="btn  text-2 text-black border-r-3">
+                    let's start Quiz </a>
+            </div>
+        </section>
+
+    </main>
+    <div class="divider-2"></div>
+
```
Comment (Developers-codz) [inline pages/rules.html:67]: remove extra spaces

**Review by Developers-codz — APPROVED** [review]
Looking good to me

File: index.html [hunk 3]
```diff
@@ -48,115 +48,123 @@ <h1 class="text-shadow"> PISTON <span class="text-accent">Quiz</span></h1>
...
-        <div class="card-vertical border-r-3 card-shadow padding-2">
-
-            <div class="card-quiz-image">
-                <img src="./assets/yezdi.jpg" alt="A GTR" class="border-r-3">
-            </div>
-            <div class="card-vertical-text">
-                <h3 class="text-center">Do you know <strong class="text-accent">YEZDI</strong> ? </h3>
+    <section class=" flex-center-space-even padding-2 flex-wrap" id="question-tiles" >
+
+<a href="./pages/rules.html">
```
Comment (sainath7878) [inline index.html:133]: Indentation is not proper.

File: pages/rules.html (hunk 2 above)
Comment (Dev-AkshatGupta) [reply pages/rules.html:67]: Ok will improve the indentation . Thanks for reviewing .

File: index.html (hunk 3 above)
Comment (Dev-AkshatGupta) [reply index.html:133]: Yupp gonna make it proper

File: css-files/root.css [hunk 4]
```diff
@@ -27,6 +27,7 @@ background-color: var(--VIOLET-50);
 }
 a{
     text-decoration: none;
+    color:#000000;
```
Comment (sainath7878) [inline css-files/root.css:30]: use variables

Comment (Dev-AkshatGupta) [reply css-files/root.css:30]: Ok I will change it to that. Thanks

**Pull request merged**